examples/demo.css

```diff
@@ -256,3 +255,10 @@ a.name-header{
     align-content: center;
     justify-content: center;
 }
+.panel-footer .insert-step{
+  margin-left:10px;
+  margin-top:-40px;
+  border-radius:6px;
```

Comment:
Hmm, shouldn't we be able to rely on the pure Bootstrap styles instead of setting these?